index.html

```diff
@@ -262,13 +262,34 @@ <h2>Implementation Details</h2>
               A list of tag files that MAY be included in a conformant Bag. Entries are either full path names relative to the bag base directory or path name patterns in which asterisks can represent zero or more characters (c.f. <a href="http://man7.org/linux/man-pages/man7/glob.7.html">glob(7)</a>).
             </p>
             <p>
-              If `Tag-Files-Allowed` is not provided, its value is assumed to be `['*']`, i.e. all tag files are allowed.
+              Conformants bags MUST NOT contain any payload that do not match the specified `Tag-Files-Allowed`. If `Tag-Files-Allowed` is not provided, its value is assumed to be `['*']`, i.e. all tag files are allowed.
```

Comment:
Not sure I agree with this. I want to allow certain files, but if they aren't there that is fine. If they are required then that should be covered by the `Tag-Files-Required`

Reply:
Nevermind, I understand what you are saying now. If a file is present it must match one of the `Tag-Files-Allowed` patterns. but if the file is absent that is fine.